README.md

```diff
@@ -45,15 +42,3 @@ yarn
 * `yarn test:contracts` - Test only contracts
 * `dev:contracts` - Set up a development environment for contracts. Watch file changes and rerun tests.
 
-
-
-
-## Deployment
-
-The deployment process focused on implementing GitOps practices. Basically, it means all master (main) builds go to production directly. For a web application, PRs also deploy to a preview environment.
```

Comment:
Maybe left at least that we deploy all changes to sepolia preview?

Reply:
i'll return and extend this section with details after i fix the CI (if you don't mind)